packages/frontend/src/pages/home/hero/index.tsx

```diff
@@ -25,28 +26,53 @@ const plusIconStyles = cva(["invisible", "md:visible", "absolute"], {
 
 export const Hero = () => {
     const { t } = useTranslation();
+    const { loading, kpiTokens } = useFeaturedKPITokens();
 
     return (
-        <div className="relative bg-orange bg-grid-light">
-            <div className="relative space-y-12 pt-7 pb-16 md:pt-24 md:pb-32">
-                <Typography
-                    variant="h1"
-                    className={{
-                        root: "px-6 md:px-10 lg:px-32 dark:text-black",
-                    }}
-                >
-                    {t("home.featuredCampaigns")}
-                </Typography>
-                <CardHorizontal className="px-6 md:px-10 lg:px-32 dark">
-                    <FeaturedCampaigns />
-                </CardHorizontal>
-                <div className="px-6 md:px-10 lg:px-32 flex flex-col space-x-0 md:space-x-8 space-y-4 md:space-y-0 md:flex-row">
-                    <Button variant="primary" size="big">
-                        <Link to="/campaigns">{t("campaign.all")}</Link>
-                    </Button>
-                    {!__PREVIEW_MODE__ && <CreateCampaignButton />}
+        <div className="relative bg-orange bg-grid-light min-h-[65vh]">
+            {loading ? (
+                <div className="flex justify-center items-center w-full h-full">
+                    <Loader />
                 </div>
-            </div>
+            ) : kpiTokens.length === 0 ? (
+                <div className="px-6 md:px-14 lg:px-36 relative space-y-12 pt-7 pb-16 md:pt-24 md:pb-32">
+                    <div className="flex flex-col md:flex-row justify-around items-center gap-10 md:gap-0">
+                        <div className="flex flex-col items-center md:items-start gap-10 w-full md:w-2/5">
+                            <Typography variant="h1">
+                                Reach your goals with a Carrot
+                            </Typography>
+                            <Typography>
+                                Easy and powerful tool to create conditional
```

Comment:
Should we add this to the `i18n`?

Reply:
Absolutely, addressed in 3325d23eb9a0cb85f7f78f3ae608c7d154bce3d3